freqtrade/freqai/prediction_models/LightGBMRegressor.py

```diff
@@ -28,15 +28,16 @@ def fit(self, data_dictionary: dict, dk: FreqaiDataKitchen, **kwargs) -> Any:
         :param dk: The datakitchen object for the current coin/model
         """
 
+        X = data_dictionary["train_features"]
```

Comment:
Modifying variable names for unknown reasons does not constitute a change. Remove this file from PR.